src/themes/teams/components/List/listItemStyles.ts

```diff
@@ -1,12 +1,21 @@
 import { pxToRem } from '../../../../lib'
 import { IComponentPartStylesInput, ICSSInJSStyle } from '../../../../../types/theme'
 import { IListItemProps } from '../../../../components/List/ListItem'
+import isFromKeyboard from '../../../../lib/isFromKeyboard'
 
 const listItemStyles: IComponentPartStylesInput<IListItemProps, any> = {
   root: ({ props: { selection, important } }): ICSSInJSStyle => ({
     ...(selection && {
       position: 'relative',
 
+      ':focus': {
+        ...(isFromKeyboard && {
+          background: 'rgba(98, 100, 167, .8)',
+          color: '#fff',
```

Comment:
we should use variables for this; steps:
- look in `siteVariables.ts` and find the colors there, for `#fff` it's `white`;
- go to `listItemVariables.ts` and define your variables there, like:
```typescript:
contentColorFocus: siteVars.xxx,
backgroundColorFocus: siteVars.white,
```
- add `variables` prop to the argument of `root` slot in this file on line 7 and replace the hardcoded colors with the defined variables